apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/hentinformasjon/BeregningKlient.kt

```diff
@@ -40,10 +40,11 @@ class BeregningKlient(config: Config, httpClient: HttpClient) {
                 failure = { errorResponse -> throw errorResponse },
             )
         } catch (e: Exception) {
-            throw BeregningKlientException(
+            logger.error(
```

Comment:
Dette er litt hacky, helst skulle eg ha skrive om hent beregning i beregnings-appen til å returnere no content, men usikker på kva konsekvensane av det er, så usikker på om eg tør.

Reply:
Jeg ser ikke helt behovet for at denne skal svelge feil -- vet vi ikke i generering av brevdata om vi trenger utbetaling eller ikke?

Reply:
Jo, men vi veit ikkje om forrige behandling hadde utbetaling eller ikkje. Det kan vera revurdering av opphør, eller innvilgelse, det veit vi per no ikkje herifrå.